l10n_it_account_tax_kind/model/account_tax_kind.py

```diff
@@ -7,11 +7,21 @@
 class AccountTaxKind(models.Model):
 
     _name = 'account.tax.kind'
-    _rec_name = 'display_name'
 
     code = fields.Char(string='Code', size=3, required=True)
     name = fields.Char(string='Name', required=True)
-    display_name = fields.Char(string='Name', compute='_compute_display_name')
+    display_name = fields.Char(string='Name', compute='_compute_display_name',
+                               store=True)
+
+    @api.multi
+    def name_get(self):
+        res = []
+        for item in self:
+            name = item.name
+            if item.display_name:
+                name = item.display_name
+            res.append((item.id, name))
+        return res
```

Comment:
@jackjack82 why both ```display_name``` and ```name_get```?  Isn't ```name_get``` sufficient?

Reply:
@jackjack82 restore the old structure for the name of the tax: it should be `[code] name`.
Moreover you can simplify everything following [this](https://github.com/odoo/odoo/blob/3656999f5e51658e0611af100c1507abc7647d94/odoo/addons/base/ir/ir_model.py#L711) example